configure.ac

```diff
@@ -0,0 +1,187 @@
+fi
+
+ac_node_version="10"
+
+if test -x "$NODE"; then
+  AC_MSG_CHECKING(make sure we have at least node ${ac_node_version})
+  case `$NODE --version` in
+  v10*)
+    AC_MSG_RESULT(ok v10)
+    ;;
+  v11*)
+    AC_MSG_RESULT(ok v11)
+    ;;
+  v12*)
+    AC_MSG_RESULT(ok v12)
+    ;;
+  v13*)
+    AC_MSG_RESULT(ok v13)
+    ;;
+  *)
+    AC_MSG_RESULT(no);
+    NODE=old-node
+  esac
+fi
+
+
+AC_PATH_PROG(OPENSSL, openssl, no)
+if  test -x "$OPENSSL"; then
+:
+else
+  AC_MSG_ERROR(could not find openssl binary)
+fi
+
+AC_MSG_CHECKING(is perl reasonably complete)
+if $PERL -MExtUtils::MakeMaker -e '' 2>/dev/null; then
+  AC_MSG_RESULT(yes. ExtUtils::MakeMaker is available);
+else
+  AC_MSG_RESULT(no)
+  AC_MSG_ERROR([a complete perl ('perl-core' in the redhat world) installation is required])
+fi
+
+AC_MSG_CHECKING([if we require a c compiler to get perl modules compiled])
+if $PERL -MIO::Socket::IP -e 'exit($IO::Socket::IP::VERSION >= 0.37)' 2>/dev/null; then
+    AC_MSG_RESULT(no)
+else
+    AC_MSG_RESULT(yes)
+    perl_cc=`$PERL -MConfig -e 'print $Config{cc}'`
+    AC_PATH_PROG(PERL_CC_PATH, $perl_cc, no)
+    AC_MSG_CHECKING(is perls favorite c compiler ($perl_cc) available)
+    if test x$PERL_CC_PATH = xno; then
+      AC_MSG_RESULT(no)
+      AC_MSG_ERROR([perl needs the '$perl_cc' compiler package to build dependencies])
+    else
+      AC_MSG_RESULT(yes)
+    fi
+fi
+
+
+AC_PROG_GREP
+
+AC_ARG_VAR(GMAKE,   [Path to local GNU Make binary])
+AC_PATH_PROGS(GMAKE, [gnumake gmake make])
+
+AC_MSG_CHECKING([for gnu make availablility])
+if  ( $GMAKE --version 2> /dev/null | $GREP GNU  > /dev/null 2>&1 );  then
+    AC_MSG_RESULT([$GMAKE is GNU make])
+else
+    AC_MSG_ERROR([GNU make not found. Try setting the GMAKE environment variable.])
+fi
+
+AC_ARG_ENABLE(pkgonly,
+        AC_HELP_STRING([--enable-pkgonly],
+                        [Skip all checking]))
+AC_SUBST(enable_pkgonly)
+
+actual_prefix=$prefix
+if test "$actual_prefix" = "NONE"; then
+    actual_prefix=$ac_default_prefix
+fi
+
+AC_MSG_CHECKING([Checking Install Prefix])
+if test -f "$actual_prefix/configure"; then
+    AC_MSG_ERROR([The install prefix must NOT point to the directory where you unpacked the source.])
+else
+    AC_MSG_RESULT([install Prefix is fine])
+fi
+
+
+AC_ARG_VAR(PERL5LIB,   [Colon separated list of perl library directories])
+AC_SUBST(PERL5LIB)
+
+AC_CONFIG_FILES([
+    Makefile
+    etc/Makefile
+    lib/Makefile
+    bin/Makefile
+    thirdparty/Makefile
+])
+
+AC_SUBST(VERSION)
+
+AC_OUTPUT
+
+
+if test "$mod_ok" = "0"; then
+    cat <<NOTES
+
+** SOME PERL MODULES ARE MISSING ******************************
+
+If you know where perl can find the missing modules, set
+the PERL5LIB environment variable accordingly. Otherwise
+just proceed and the missing modules will be installed
+automatically.
+
+To force the system to download missing perl modules, you can
+do a `touch PERL_MODULES` before running `$GMAKE`
+NOTES
+
+fi
```

Comment:
this is an artefact from a time when I tested for the presence of perl modules in the configure script